beetmoverscript/script.py

```diff
@@ -30,16 +31,30 @@
     context.task = get_task(context.config)  # e.g. $cfg['work_dir']/task.json
     # 2. validate the task
     validate_task_schema(context)
-    # 3. generate manifest
+    # 3 grab manifest props with all the useful data
+    context.properties = await get_props(context)
```

Comment:
Heh :) Would you believe me if I told you that initially that was my workaround? That was my motivation as well and it was all working fine but then I ran the PR on a short feedback session with async-@rail who asynced everything and passed me the virus to async everything 😃 

On a more serious note, your second statement is the motivation that stands behind async-ing that call there. I added requests as dep and used requests.get there before spotting the retry_request in scriptworker and async-ing the stuff.